packages/data-stores/src/plans/test/resolvers.ts

```diff
@@ -30,6 +13,8 @@ jest.mock( 'wpcom-proxy-request', () => ( {
 	__esModule: true,
 } ) );
```

Comment:
I decided not to mock the `wpcom-request-controls` module (see [commit](https://github.com/Automattic/wp-calypso/pull/50492/commits/0101d833745fa872724e458401b66b992c52349d)), as those functions are still part of the `data-stores` package.

I know that by not mocking these functions, these unit tests become also partially integration tests — but I think it's fine in this case.

Reply:
We probably have to update our resolvers somewhere in the future (https://github.com/Automattic/wp-calypso/pull/40371#issuecomment-787843426) since `wpcom-request-controls` breaks when we migrate to OAuth.

Reply:
That's correct, but I would wait for directions from the Calypso folks before making any changes to that part of the `data-stores` package